test/unit/models/PackageSpec.scala

```diff
@@ -28,55 +28,55 @@ import uk.gov.hmrc.accessibilitystatementfrontend.models.{prettyPrintDate, repor
-      val mcc = app.injector.instanceOf[MessagesControllerComponents]
-      val requestDefaultLanguage = FakeRequest()
+      val app                         = new GuiceApplicationBuilder().build()
+      val mcc                         = app.injector.instanceOf[MessagesControllerComponents]
+      val requestDefaultLanguage      = FakeRequest()
       implicit val messages: Messages = mcc.messagesApi.preferred(requestDefaultLanguage)
 
-      val firstDate = new GregorianCalendar(2020, Calendar.FEBRUARY, 28).getTime
+      val firstDate  = new GregorianCalendar(2020, Calendar.FEBRUARY, 28).getTime
       val secondDate = new GregorianCalendar(2020, Calendar.MARCH, 15).getTime
-      val thirdDate = new GregorianCalendar(2020, Calendar.MAY, 1).getTime
+      val thirdDate  = new GregorianCalendar(2020, Calendar.MAY, 1).getTime
 
-      prettyPrintDate(firstDate) should equal("28 February 2020")
+      prettyPrintDate(firstDate)  should equal("28 February 2020")
       prettyPrintDate(secondDate) should equal("15 March 2020")
-      prettyPrintDate(thirdDate) should equal("01 May 2020")
+      prettyPrintDate(thirdDate)  should equal("01 May 2020")
     }
 
     "return the date as a string in the format 01 January 2020 in English if language cookie set to en" in {
-      val app = new GuiceApplicationBuilder().build()
-      val mcc = app.injector.instanceOf[MessagesControllerComponents]
-      val requestEnglishLanguage = FakeRequest().withCookies(Cookie("PLAY_LANG", "en"))
+      val app                         = new GuiceApplicationBuilder().build()
+      val mcc                         = app.injector.instanceOf[MessagesControllerComponents]
+      val requestEnglishLanguage      = FakeRequest().withCookies(Cookie("PLAY_LANG", "en"))
       implicit val messages: Messages = mcc.messagesApi.preferred(requestEnglishLanguage)
 
-      val firstDate = new GregorianCalendar(2020, Calendar.FEBRUARY, 28).getTime
+      val firstDate  = new GregorianCalendar(2020, Calendar.FEBRUARY, 28).getTime
       val secondDate = new GregorianCalendar(2020, Calendar.MARCH, 15).getTime
-      val thirdDate = new GregorianCalendar(2020, Calendar.MAY, 1).getTime
+      val thirdDate  = new GregorianCalendar(2020, Calendar.MAY, 1).getTime
 
-      prettyPrintDate(firstDate) should equal("28 February 2020")
+      prettyPrintDate(firstDate)  should equal("28 February 2020")
       prettyPrintDate(secondDate) should equal("15 March 2020")
-      prettyPrintDate(thirdDate) should equal("01 May 2020")
+      prettyPrintDate(thirdDate)  should equal("01 May 2020")
     }
 
     "return the date as a string in the format 01 January 2020 in Welsh if language cookie set to cy" in {
-      val app = new GuiceApplicationBuilder().build()
-      val mcc = app.injector.instanceOf[MessagesControllerComponents]
-      val requestWelshLanguage = FakeRequest().withCookies(Cookie("PLAY_LANG", "cy"))
+      val app                         = new GuiceApplicationBuilder().build()
```

Comment:
Can we change this to match `automatedTestingOnly = None,` from further down?

If we use one space rather than lining them up we'll have fewer changes just for formatting.

Reply:
Would it not be best to follow the platform recommended style-guide for consistency?